docs/data/toolpad/getting-started/quickstart.md

```diff
@@ -0,0 +1,150 @@
+# Quickstart
+
+Get started building a Toolpad app!
+
+<p class="description">
+This page will guide you into creating your very first Toolpad application. We will create a "hello world" type of application that will ask for a name and print a greeting.
+</p>
+
+## Prerequisites
+
+Make sure you have a running Toolpad instance.
+
+1. Download the docker compose file
+
+   ```sh
+   curl -LO https://raw.githubusercontent.com/mui/mui-toolpad/master/docker/compose/docker-compose.yml
+   ```
+
+1. Start the docker compose services
+
+   ```sh
+   docker-compose -f docker/compose/docker-compose.yml up -d
+   ```
+
+Toolpad will be accessible under `http://localhost:3000/`.
+
+## Navigating Toolpad
+
+![Toolpad app](/static/toolpad/app-ui-chrome.png)
+
+There are 4 most frequently used areas:
+
+- `Instance editor` - this is there you define and manage pages, connection configurations, custom component definitions, etc...
+- `Component's library` - drag component instance from this drawer and drop it in `Application editor` to build UI
+- `Application editor` - this is a place where you build your app and immediately preview how it looks like
+- `Properties editor` - modify properties of a different instance
+
+## Building your first application
+
+1. Open Toolpad by navigating to [http://localhost:3000/](http://localhost:3000/)
+
+   ![Apps overview](/static/toolpad/apps-overview.png)
+
+1. Click "CREATE NEW" and name your application. Confirm by clicking "CREATE"
+
+   ![Toolpad overview](/static/toolpad/step-1.png)
+
+1. Hover over `Component's library` and drag `DataGrid` and `Image` components into `Application editor`
+
+   ![Drag components](/static/toolpad/step-2.png)
+
+   Congratulations, we are done building the UI! Now all we need to do is connect it with a data source.
+
+1. Click anywhere inside `Application editor` (except on the components that you just added) to deselect added components
+
+1. Locate `ADD QUERY` button inside `Properties editor` and press that to start configuring our data source
+
+   ![Add query](/static/toolpad/step-3.png)
+
+1. This time we are going to use simple `FETCH` type to query our data. Choose and press `CREATE QUERY`
+
+   ![Fetch type](/static/toolpad/step-4.png)
+
+1. Let's fetch some data about dogs from `https://dog.ceo/dog-api` (if you're more of a cat person you could use cat data API from `https://catfact.ninja`. We apologize in advance - following tutorial steps are more suited for the dog data API but should be easily adaptable for cats as well!)
```

Comment:
Do we want to rely on these sort of APIs that we don't control? Perhaps we should just upload a static JSON with dummy data to github and fetch it through `https://raw.githubusercontent.com/`?

Reply:
I've removed sentence with cat's api ref. I'd like to keep dogs example for now as it requires to update multiple screenshots as well. I will add a follow up task to change this to something that we can control easier 👍